alg/viewshed/viewshed.cpp

```diff
@@ -188,6 +188,18 @@ GDALDatasetH GDALViewshedGenerate(
     oOpts.invisibleVal = dfInvisibleVal;
     oOpts.outOfRangeVal = dfOutOfRangeVal;
 
+    const char* pszStartAngle = CSLFetchNameValue(papszExtraOptions, "START_ANGLE");
```

Comment:
please document those options at line 109

and fix the code formatting by setting up pre-commit as indicated in https://gdal.org/en/stable/development/dev_practices.html#commit-hooks